Libraries/CustomComponents/NavigationExperimental/NavigationCardStackPanResponder.js

```diff
@@ -131,7 +131,7 @@ class NavigationCardStackPanResponder extends NavigationAbstractPanResponder {
       */
       props.gestureResponseDistance || 30;
 
-    if (positionMax != null && currentDragPosition > positionMax) {
+    if (positionMax != null && currentDragPosition - RESPOND_THRESHOLD > positionMax) {
```

Comment:
Could you add a comment please? The PR description is good but when people read the code they won't see the PR description and to me it would be very non-obvious why we subtract `RESPOND_THRESHOLD` here.